src/app/store/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,58 @@
+<div fxLayout="row">
+  <mat-card fxFlex="60" fxFlex.md="100" fxFlex.sm="100" fxFlex.xs="100">
+    <mat-spinner *ngIf="loading"></mat-spinner>
+    <div fxLayout="row" fxLayoutAlign="center">
+    <img mat-card-image [src]="product.imageUrl" alt="{{ product.name }}" >
```

Comment:
this page assumes that product will not be null, would be a good idea to wrap the content in an ngIf = product and add some text to display when ngIf = !product

Reply:
I will display the error message returned from the api if it doesn't return a product.